ci-operator/step-registry/ipi/install/install/ipi-install-install-ref.yaml

```diff
@@ -6,5 +6,8 @@ ref:
     requests:
       cpu: 1000m
       memory: 2Gi
+  env:
+  - name: OPENSHIFT_UPGRADE
+    default: "false"
```

Comment:
Please add [`documentation`][1] :).  Something like:

> Select between ${RELEASE_IMAGE_INITIAL} (true) or ${RELEASE_IMAGE_LATEST} (false) for the installed version.

Not clear to me if `documentation` supports Markdown.  If so, backtick the environment variables?

[2]: https://github.com/openshift/ci-tools/pull/854/files#diff-95aa6a9a4f1daf516567fb2eb073cb3fR1225-R1226